README.md

```diff
@@ -40,7 +48,4 @@ To stop the script, type `window.clearTimeout(thingTimer);` into the console and
 	 - Morale Booster if available and lane has > 2 live enemies
 	 - Decrease Cooldowns if another player used a long-cooldown ability < 10 seconds ago
 	
-- purchase abilities and upgrades intelligently
-
-- automatically update the script by periodically checking https://raw.githubusercontent.com/mouseas/steamSummerMinigame/master/autoPlay.js
-     - Simplify the Greasemonkey/Tampermonkey code so it just loads from ^. Add a separate .js file for this purpose, and obviously update the README.
\ No newline at end of file
+- purchase abilities and upgrades intelligently
```

Comment:
My apologies. At the time it looked like the first one was for GM/TM, when in fact it is for the manual implementation.

With the updated instructions, Tampermonkey and Greasemonkey both load from `https://raw.githubusercontent.com/mouseas/steamSummerMinigame/master/autoPlay.js` instead of copy and pasting the code now. The README has been updated, so I'm not sure why the last TODO is necessary. Is there any other reason?